tests/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN env
 RUN apt-get update -y
 RUN apt-get install -y \
             git \
-            python3 \
```

Comment:
I have already changed `Dockerfile` in [this branch](https://github.com/nutti/fake-bpy-module/blob/update_docker_image/tests/Dockerfile) (will be deleted after merge).
I think this patch is not needed.

Reply:
I saw it. The main reason why I still pushed it was a few improvements from me:
- pip installation seemed too complex
- removed links to `pip` and `python` binary
- using new `PYTHON_BIN` env variable

Reply:
I understood your ideas.  
But sorry, I would like to use my script because of below reasons.

* Sometime mistake the command which is linked to python2.
* New environment variable should not be introduced which is not required.

Reply:
BTW, newly created image is now uploading to Docker Hub with using my Dockerfile before your PR is published.
Personally, it is good to consistent between Dockerfile and Docker image.

Sorry for late announcement about this.

Reply:
> Sorry for late announcement about this.

No worries

> Sometimes mistake the command which is linked to python2.

I understand, but if you link python3 -> python, then you are just hiding the errors in case someone forgets to use `${python}` in the bash scripts.

> New environment variable should not be introduced which is not required.

I introduced `PYTHON_BIN`, so people can build the module locally. Before my changes, it was very hard to build it on any Linux machine.

If you don't want to to `ENV PYTHON_BIN python3.7` and you want to link `python`,  then we should set `ENV PYTHON_BIN python`, to be explicit on the CI. I opened a bug report here #37

Reply:
I understood your intention.
Let us discuss about this issue at #37 .